packages/gatsby-theme-newrelic/src/components/CodeEditor.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ClassNames } from '@emotion/core';
+import Editor from 'react-simple-code-editor';
+import CodeHighlight from './CodeHighlight';
+
+const CodeEditor = ({ value, language, lineNumbers, onChange }) => {
+  const lineNumberWidth = value.trim().split('\n').length.toString().length;
+
+  return (
+    <ClassNames>
```

Comment:
ooh I didn't know this was an option with emotion, interesting

Reply:
Its only really needed in 1 case. The `css` API expects you to be adding it to the `className` prop to work its compile-time magic. In this case, I needed the apply the class name to a [`textareaClassName`](https://github.com/newrelic/gatsby-theme-newrelic/pull/44/files#diff-cc5dba3515fd332c99ba16b61136dd3dR32), so this utility is provided to be able to make the `css` function return a stringified classname that I can pass to other props. 

Its rare you'll need this, but good to know it exists 🙂